python/vowpalwabbit/pyvw.py

```diff
@@ -2,12 +2,35 @@
 """Python binding for pylibvw class"""
 
 from __future__ import division
-import pylibvw
+from pylibvw import example as pylibvw_example, vw as pylibvw_vw
```

Comment:
You must also import predictor and search.

This is technically changing the public interface of the package, so I am hesitant to merge this atm.

Reply:
I suggest to import it only when needed. It is because it creates a significant time difference in importing the module

Reply:
Does this break the public interface? Or is it still possible to do the below?
```
from vowpalwabbit import pyvw
pyvw.pylibvw.search...
```

Reply:
It doesn't break the public interface. Just I meant to import required classes from pylibvw. Other functionalities remains the same as they are. Yes, it will work

Reply:
Ahha, I wonder how will `pyvw. pylibvw.search` work. As there is no such class/function named `pylibvw`. So, I think this should work presently too. I am not aware if python allows this

Reply:
> Does this break the public interface? Or is it still possible to do the below?

This won't work even in current master.
```

>>> from vowpalwabbit import pyvw
>>> pyvw.pylibvw.vw
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'vowpalwabbit.pyvw' has no attribute 'pylibvw'
```